pnpm-lock.yaml

```diff
@@ -6110,6 +6142,20 @@ packages:
     engines: {node: '>= 0.10.0'}
     dev: true
 
+  /ember-functions-as-helper-polyfill@2.1.2(ember-source@4.8.6):
+    resolution: {integrity: sha512-yvW6xykvZEIYzzwlrC/g9yu6LtLkkj5F+ho6U+BDxN1uREMgoMOZnji7sSILn5ITVpaJ055DPcO+utEFD7IZOA==}
+    engines: {node: '>= 14.0.0'}
+    peerDependencies:
+      ember-source: ^3.25.0 || >=4.0.0
+    dependencies:
+      ember-cli-babel: 7.26.11
```

Comment:
ember-cli-babel 8 (latest) is required


You may want to try bumping

Reply:
Tho, this polyfill shouldn't be required for the replo, yeah?

Reply:
We're on     `"ember-cli-babel": "^8.2.0",`